src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --focus_color: #00acdc;
+}
+
+html {
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.main_header {
+  display: flex;
+  width: 100%;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
+  padding-inline-start: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.list__item:nth-child(n + 2) {
```

```suggestion
.list__item:not(:first-child) {
```
It's actually the same but looks more understandable in my opinion. Not so important, just want to mention it.